docs/user-guide/ze-install-configuring-ze.md

```diff
@@ -79,3 +79,70 @@ To define the level of detail included in log files:
 2. Select the **User** or **Workspace** tab, depending on the settings you want to update.
 3. In the Settings navigation menu, open the **Extensions** menu and click **Zowe Explorer**.
 4. In the **Logger** section, open the drop-down menu to select a different detail setting.
+
+## Modifying the default sort order for data sets and jobs
+
+To change the default sort order for data sets and jobs:
+
+1. In Zowe Explorer settings, scroll to a data set or job setting type.
+2. Click the setting's corresponding **Edit in settings.json** link.
+3. This opens the `settings.json` file in an **Editor** tab.
+
+![Default Data Set Sort Settings](../images//ze/ZE-default-ds-sort-setting.png)
+![Default Job Sort Settings](../images//ze/ZE-default-job-sort-setting.png)
+
+:::tip
+
+Alternately, on the VS Code Command Palette (`Ctrl`/`Cmd` + `Shift` + `P`), enter **Preferences: Open User Settings (JSON)** to display the Settings editor.
+
+:::
+
+The following allowed directions are available for sorting both data sets and jobs:
+
+| Direction | Description |
+| --- | --- |
+| `Ascending` | Sorts in ascending order. |
+| `Descending` | Sorts in descending order. |
+
+The following allowed methods are available for sorting data sets:
+
+| Method | Description | 
+| --- | --- |
+| `Name` | Sorts by the name of the data set. |
+| `DateCreated` | Sorts by the date the data set was created. |
+| `LastModified` | Sorts by the date the data set was last modified. |
+| `UserId` | Sorts by the user ID who last modified the data set. |
+
+The following allowed methods are available for sorting jobs:
+
+| Method | Description |
+| --- | --- |
+| `Id` | Sorts by the job ID. |
+| `DateSubmitted` | Sorts by the date the job was submitted. |
+| `DateCompleted` | Sorts by the date the job was completed. |
+| `Job Name` | Sorts by the name of the job. |
```

Comment:
It looks like the `Method` for `Job Name` has a space between the words while the others are in pascal case - is this intentional?

Reply:
Not intentional, thanks!